src/data/nat/prime.lean

```diff
@@ -633,6 +633,14 @@ lemma mem_factors_iff_dvd {n p : ℕ} (hn : 0 < n) (hp : prime p) : p ∈ factor
 ⟨λ h, prod_factors hn ▸ list.dvd_prod h,
  λ h, mem_list_primes_of_dvd_prod hp (@prime_of_mem_factors n) ((prod_factors hn).symm ▸ h)⟩
 
+lemma dvd_of_mem_factors {n p : ℕ} (h : p ∈ n.factors) : p ∣ n
+  :=
```

Comment:
Mathlib style is:
```suggestion
lemma dvd_of_mem_factors {n p : ℕ} (h : p ∈ n.factors) : p ∣ n :=
```

Reply:
(this applies throughout)

Reply:
Ah, sorry, I knew this and failed to spot it.  I've gone through and fixed this throughout.